services/workflows-service/src/rule-engine/core/condition/enums.ts

```diff
@@ -0,0 +1,14 @@
+export enum ConditionType {
+  EQUALS = 'EQUALS',
+  BETWEEN = 'BETWEEN',
+  GT = 'GT',
+  LT = 'LT',
+  GTE = 'GTE',
+  LTE = 'LTE',
+  LAST_YEAR = 'LAST_YEAR',
+}
+
+export enum ConditionOperator {
+  AND = 'and',
+  OR = 'or',
+}
```

Comment:
Enums in TS are considered a bad practice. You can use a simple object instead. https://www.youtube.com/watch?v=jjMbPt_H3RQ
You can check how we implemented `CaseState` as a good example.